app/scripts/app.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+/**
+ * @ngdoc overview
+ * @name bitcoin-price-demo
+ * @description
+ * # bitcoin-price-demo
+ *
+ * Main module of the application.
+ */
+var myapp = angular.module('bitcoin-price-demo', [
+  'ngAnimate',
+  'ngAria',
+  'ngCookies',
+  'ngMessages',
+  'ngResource',
+  'ngRoute',
+  'ngSanitize',
+  'ngTouch',
+  'ui.bootstrap',
+  'nvd3'
+]);
+myapp.config(function ($routeProvider) {
+  $routeProvider
+    .when('/', {
+      templateUrl: 'views/main.html',
+      controller: 'MainCtrl'
+    })
+    .when('/about', {
+      templateUrl: 'views/about.html',
+      controller: 'AboutCtrl'
+    })
+    .otherwise({
+      redirectTo: '/'
+    });
+});
+// http://stackoverflow.com/questions/17144180/angularjs-loading-screen-on-ajax-request
```

Comment:
i would put the directives into seperate directories and files
e.g. `scripts/directives/loading/loading.js`